### PR TITLE
Added type for website, blogpost and podcast

### PR DESCRIPTION
**Pull request by rm2342**
(as part of "article-newspaper")

### CODE REVIEW

File: neue-juristische-wochenschrift.csl [hunk 1]
```diff
@@ -109,7 +109,7 @@
             <text macro="firstpage-locator"/>
           </group>
         </if>
-        <else-if type="article-newspaper">
+        <else-if type="article-newspaper post-weblog webpage song">
```
Comment (adam3smith) [inline neue-juristische-wochenschrift.csl]: this needs match="any" (if/else-if default to match="all" which would never evaluate to true in this case.)

Comment (rm2342) [inline neue-juristische-wochenschrift.csl]: Thank you. Funnily, it worked fine without the match="any". I added it
anyway, of course.

**Comment by adam3smith** [comment]
Thanks. Yes, Zotero may still be too forgiving here, but other CSL implementations (e.g. pandoc's) will stumble over this.

**Comment by rm2342** [comment]
Thank you. I will update my other styles in the same manner in the near
future (wanted to run one style through the process first).

**Pull request merged**